perfrunbook/system-load-and-compute-headroom.md

```diff
@@ -1,120 +1,68 @@
-148205, 21, 975
-197231, 31, 918
-247118, 35, 960
-297056, 41, 957
-345138, 50, 1037
-395424, 60, 1037
-443433, 70, 1106
-493272, 81, 1173
-543327, 88, 1301
-591411, 94, 1806
-
-```
-
+644506 packet/s would be expected as maximum throughput but 736860 packets/s where handled, thus 14% better than expected.
 
 
 ![](images/system-load/c6i.png)
 
-The Intel SMT system degrades when loaded close to 100%. It achieves 86% of the performance it should have shown based on the 50% number: 345138/0.50=690276 packet/s would be expected. Thus 591411/690276=0.857 of the predicted value is achieved.
+The Intel SMT system degrades when loaded close to 100%. It achieves 86% of the performance it should have shown, based on the 50% number: 690276 packet/s would be expected, 591411 packets/s where actually handled.
 At the 50% CPU load mark, a c6i can handle an additional 71% of the traffic it did up to 50% (345k,246k), whereas the c6g is able to serve another 130% of that (314k,423k).
+The c6i would need 2 additional vCPUs to be on par with c7g in packets/s.
 
-#### Third Experiment
-
-These are the results of a hashing benchmark that initially favor c6i instances:
-
-![](images/system-load/c7g-c6g-vs-c6i.png)
-
-Of interest here is that the initial c6i advantage over c7g starts to disappear beyond 50% system load. Again, because the Linux kernel will prefer using cores first and resort to SMT threads after 50% load, c6i performance decreases from thereon. The c6g and c7g instances performance increases linearly. Here,  a c7g instance performs at the same level as c6i at 100% CPU load.
-
-#### Discussion
-
-In the first experiment, for the c6i to maintain the same response time as the c7g, it would need more vCPUs. It can stay on par until 5 of the 8 vCPUs are used. 
-In the second experiment, the c6i would need 2 additional vCPUs to be on par with c7g in packets/s.
-The third experiment shows that the c6i has an advantage in processing speed when the system is loaded below 50%. 
-Experiment one and two also indicate this, if not as clearly.
-
-
-#### Conclusions
+#### Conclusion
 
-Graviton instances compute performance increases linear with CPU load, x86 performance increases less after 50% CPU load. This is because x86-type CPUs employ symmetric multithreading, aka. 'Hyperthreading'. Based on the above, load balancer thresholds can, in many cases, be set higher on Graviton instances than on x86-type instances and thus lead to significant savings in the size of the required server fleet, as the Netty example shows. 
+Graviton instances compute performance increases near-linearly with CPU load, x86 performance increases less after 50% CPU load. This is because x86-type CPUs employ symmetric multithreading, aka. 'Hyperthreading'. Based on the above, load balancer thresholds can, in many cases, be set higher on Graviton instances than on x86-type instances and thus lead to significant savings in the size of the required server fleet, as the Netty example shows.
```

Comment:
This is because x86-type CPUs -> This is because our x86 based EC2 instances employ

Reply:
done